pg_sequencer.gemspec

```diff
@@ -17,9 +17,9 @@ Gem::Specification.new do |s|
   s.test_files = Dir["test/**/*"]
 
   # s.add_dependency "rails", "~> 3.1.0"
-  s.add_dependency 'activerecord', '>= 3.1.0'
+  s.add_dependency 'activerecord', '>= 5.1.0'
```

Comment:
I don't see any changes specific to activerecord, so looks like it can work with previous versions of AR, no need to do >= 5.1 I think